static/index.html

```diff
@@ -369,16 +369,20 @@ <h5>
                         </a>
                     </li>
                     <li>
-                        <a class="grey-text text-lighten-3 waves-effect" target="_blank" href="https://www.youtube.com/channel/UCKNtEpIxdgpdhglhVob3NfA">
+                        <a class="grey-text text-lighten-3 waves-effect" target="_blank" href="https://www.youtube.com/c/ImpactClient/">
                             <h5>
                                 <i class="medium fa fa-youtube-square"></i> YouTube
                             </h5>
                         </a>
                     </li>
                     <li>
-                        <a class="grey-text text-lighten-3 waves-effect" target="_blank" href="https://twitter.com/ImpactClient">
+                        <a class="grey-text text-lighten-3 waves-effect" target="_blank" href="https://twitter.com/Impact_Client">
```

Comment:
Why this change? Who controls `@ImpactClient` vs `@Impact_Client`?

Reply:
think a few staffs got bored of bradys ImpactClient doing nothing so someone made Impact_Client lol